cmd/openfga/openfga_test.go

```diff
@@ -11,8 +11,8 @@ import (
 	"time"
 
 	"github.com/cenkalti/backoff/v4"
-	"github.com/openfga/openfga/mocks"
 	"github.com/openfga/openfga/pkg/logger"
+	"github.com/openfga/openfga/server/authentication/mocks"
```

Comment:
Why not leave this under the top-level (shared) mocks package? It will make it easier to skip package coverage on all mocks (generated and manually defined).

Reply:
I tried to keep the mocks together with the code that they are mocking, but this works too.

Reply:
Actually.. 

 > you'll only have to ignore the mocks directory

i'm ignoring the `mocks` pattern in directories, so 🤷‍♀️